app/i18n/global-messages.js

```diff
@@ -300,7 +301,10 @@ export const environmentSpecificMessages = {
   }),
 };
 
-export function listOfTranslators(contributorsList, contributorsAck): string {
+export function listOfTranslators(
```

Comment:
this is not from this PR, but it’s kind of odd to call something a list when is a string 🤔